src/components/UserAvatar/UserAvatar.tsx

```diff
@@ -96,15 +95,14 @@ const UserAvatar: FC<UserAvatarProps> = forwardRef<
   ) =>
     any(isNotUndefined, [href, to, onClick]) ? (
       <RootUserAvatar
-        ref={ref}
+        ref={ref as unknown as Ref<HTMLButtonElement>}
```

Comment:
why do we need this cast ?

Reply:
<img width="1609" alt="Screenshot 2023-06-06 at 17 57 17" src="https://github.com/securityscorecard/design-system/assets/18654425/d7fad258-f732-4698-a8b0-9a4c6dd28879">

Reply:
I dropped the mid-conversion to `unknown`, not sure why I added it there. The casting needs to stay there because we need to add the correct type based on if the avatar is interactive or not